actool/manifest.go

```diff
@@ -250,6 +264,23 @@ func patchManifest(im *schema.ImageManifest) error {
 		}
 	}
 
+	if patchSeccompMode != "" {
```

Comment:
I was referring to the other side: if the set is not empty but the mode is not given (e.g. the user called `actool patch-manifest --seccomp-set=chroot`). In this case, the current code in `actool` would just ignore the seccomp parameter and not add any isolator. I think `actool` should return an error in that case instead of silently doing nothing.